app/views/contexts/thank_you_context.py

```diff
@@ -2,22 +2,47 @@
 from typing import Mapping
 
 from flask import url_for
+from flask_babel import format_datetime, lazy_gettext
 
 from app.data_models.session_data import SessionData
 from app.libs.utils import convert_tx_id
 from app.views.contexts.email_form_context import build_email_form_context
 
 
 def build_default_thank_you_context(
-    session_data: SessionData, submitted_at: datetime
+    session_data: SessionData, submitted_at: datetime, survey_type: str
 ) -> Mapping:
+    text = lazy_gettext("Your answers have been submitted for ")
+    submitted_on = {
+        "term": lazy_gettext("Submitted on:"),
+        "descriptions": [
+            {"description": format_datetime(submitted_at, format="dd LLLL yyyy HH:mm")}
+        ],
+    }
+    submission_reference = {
+        "term": lazy_gettext("Submission reference:"),
+        "descriptions": [{"description": convert_tx_id(session_data.tx_id)}],
+    }
+    if survey_type == "social":
+        submission_text = lazy_gettext("Your answers have been submitted.")
+        items_list = [submitted_on]
```

Comment:
It would be clearer if `items_list` was named something like `metadata_items`.

Reply:
Renamed to metadata_items. Thanks!